package.json

```diff
@@ -43,6 +45,7 @@
     "react-oncoprint": "^1.2.3",
     "react-plotly.js": "^2.2.0",
     "react-sequence-viewer": "git://github.com/plotly/react-sequence-viewer.git",
+    "scipy": "^0.1.4",
```

Comment:
Where is this package used?

Reply:
(Maybe it was an accidental `npm install scipy` instead of `pip install scipy`)

Reply:
@rpkyle ☝️

Reply:
Yep, that seems likely. Will fix!

Reply:
fixed in 56fe632